modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go

```diff
@@ -122,16 +119,16 @@ func getStatusSummary(dbCtx *dbcontext.DbContext, monitor *crd.DowntimeMonitor)
 			muteDowntimeTypes: muteTypes,
 		}
 
-		resp, err := getStatus(dbCtx, monitor, filter)
+		resp, err := getStatus(h.DbCtx, h.DowntimeMonitor, filter)
 		if err != nil {
 			log.Errorf("cannot calculate status for group %s: %v", group, err)
-			continue
+			return groupStatuses, StatusOutage, err
```

Comment:
Just a note: it returns partial groupStatuses, may be `nil` is better?

Reply:
Returning nil